### PR TITLE
Uncommented test queries in qp_misc_jiras and domain tests #3934

### PR DESCRIPTION
**Pull request by vraghavan78**

### CODE REVIEW

**Comment by HaozhouWang** [comment]
+1

File: src/test/regress/expected/domain.out [hunk 1]
```diff
@@ -22,7 +22,7 @@ create domain domainvarchar varchar(5);
 create domain domainnumeric numeric(8,2);
 create domain domainint4 int4;
 create domain domaintext text;
--- Test explicit coercions --- these should succeed (and truncate)
+-- Test explicit coercions, these should succeed (and truncate)
```
Comment (hlinnaka) [inline src/test/regress/expected/domain.out]: This is an upstream test, let's not change the spelling.

Comment (vraghavan78) [reply src/test/regress/expected/domain.out]: ok.

**Comment by vraghavan78** [comment]
Merged thanks!

**Pull request closed without merging**